doc/content/resources/multiapps_chps/chp_5_multiapps.md

```diff
@@ -0,0 +1,215 @@
+
+  2. `TIMESTEP_END`: Launch child application at the end of the time step
+
+  3. `INITIAL`: Launch child application only at the initial condition to set up the initial conditions for the solve
+
+  4. `NONLINEAR`: This option has limited applications and is used to inject the coupling information into the middle of the Newton solve which will slow down the newton convergence but could speed up the conversion of the entire system.  
+
+#### Finer Timesteps in Child Applications
+
+Applications may use different time step sizes by allowing `subcycling`. The subcycling option is useful for allowing different timescales. For instance, a relatively slow physics application may use bigger time steps (e.g. 10 sec) whereas a rapidly changing physics may use relatively small-time step (e.g. 0.1 sec). Thus, the first app takes one time-step of 10 sec while the other app takes 100 time-steps of 0.1 sec to reach the same end time as the first application.
+
+!media media/resources/advsubcycling.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id= subcycling
+      caption=Schematic drawing of the sub-cycling concept.
+
+ [subcycling] illustrates a transient Multiphysics simulation in which Griffin, Bison, and SAM are coupled at different timescales using the subcycling option. When performing sub-cycling, transferred auxiliary variables on child applications may be interpolated between the start time and the end time of the main app with the `interpolate_transfers` parameter. When using the TransientMultiApp type, the `allow_sub_cycling` parameter can be used to invoke subcycling.
+
+#### Locations of Child Applications
+
+Child applications can be launched at various spatial positions of the parent application geometry by using the 'positions' input parameter. Positions are simply a list of 3D coordinates that describe the offset of the child application's origin from the parent application's origin. If no positions are provided, the child application will be launched at position is `0 0 0` within the parent application. Notice that in the example below, the number of child applications to be invoked is not explicitly defined but rather is inferred from the number of positives given. The example shows five different positions, thus, five independent versions of the child application `BisonApp` will be created and each of them will be associated with a different position within the parent application domain. The user may also provide a position file instead of explicitly providing the positions in the MultiApp block; this feature is useful when the user wants to invoke thousands of child application as the separate input file can contain all this information without cluttering the main input file. A common use of positions in nuclear reactor analysis is the launching of individual fuel rod (BISON) simulations at various fuel rods within a reactor assembly or core.  
```

Comment:
```suggestion
Child applications can be launched at various spatial positions of the parent application geometry by using the 'positions' input parameter. Positions are simply a list of 3D coordinates that describe the offset of the child application's origin from the parent application's origin. If no positions are provided, the child application will be launched at the position `0 0 0` within the parent application. Notice that in the example below, the number of child applications to be invoked is not explicitly defined but rather is inferred from the number of positions given. The example shows five different positions, thus, five independent versions of the child application `BisonApp` will be created and each of them will be associated with a different position within the parent application domain. The user may also provide a position file instead of explicitly providing the positions in the MultiApp block; this feature is useful when the user wants to invoke thousands of child application as the separate input file can contain all this information without cluttering the main input file. A common use of positions in nuclear reactor analysis is the launching of individual fuel rod (BISON) simulations at various fuel rods within a reactor assembly or core.  
```